frontend/www/js/omegaup/ui.js

```diff
@@ -214,35 +214,28 @@ let UI = {
       success = false;
       error = data.error;
     };
-    $('input[type=checkbox]').each(function() {
-      if (this.checked) {
-        operation(this.id, resolve, reject);
-      }
-    });
-
-    // Wait for all
-    $(document).ajaxStop(function() {
-      if (!isStopExecuted) {
-        // Make sure we execute this block once. onOperationFinish might
-        // have
-        // async calls that would fire ajaxStop event
-        isStopExecuted = true;
-        $(document).off('ajaxStop');
+    let promises = [];
+    for (const item of items) {
+      promises.push(
+        new Promise(function() {
+          operation(item, resolve, reject);
+        }),
+      );
+    }
```

Comment:
si no quieres esperar a ese PR, quizás puedes intentar esto a ver si jala:

```suggestion
    const promises = items.map(item => new Promise((accept, reject) => {
       operation(item).then(data => {
         accept(data);
         return data;
       }).fail(err => {
         reject(err);
         return err;
        });
    });
```

Reply:
ah sabes qué? qué bueno que vi este cambio a tiempo porque me acabo de dar cuenta que la implementación en ese otro PR estaba toda mal!

ahora sí, acabo de mandar ese PR actualizado y de paso quité toda esta locura de mezcla entre deferreds y promesas, así que `bulkOperation` ha muerto!

Reply:
listo! dale rebase.